src/test/java/teammates/logic/core/FeedbackResponsesLogicTest.java

```diff
@@ -1712,6 +1714,123 @@ public void testGetSessionResultsForUser_orphanResponseInDB_shouldStillHandleCor
+                .withRecipientType(FeedbackParticipantType.STUDENTS_EXCLUDING_SELF)
+                .withGiverType(STUDENTS)
+                .build();
+        boolean response = FeedbackResponsesLogic.inst().isResponseOfFeedbackQuestionVisibleToStudent(question);
+        assertEquals(response, true);
+    }
+
+    @Test
+    public void testIsResponseOfFeedbackQuestionVisibleToStudentCT4(){
+
+        List<FeedbackParticipantType> showResponseTo = new ArrayList<>();
+        showResponseTo.add(RECEIVER);
+
+        FeedbackQuestionAttributes question = FeedbackQuestionAttributes.builder()
+                .withShowResponsesTo(showResponseTo)
+                .withRecipientType(FeedbackParticipantType.STUDENTS_IN_SAME_SECTION)
+                .withGiverType(STUDENTS)
+                .build();
+        boolean response = FeedbackResponsesLogic.inst().isResponseOfFeedbackQuestionVisibleToStudent(question);
+        assertEquals(response, true);
+    }
+
+    @Test
+    public void testIsResponseOfFeedbackQuestionVisibleToStudentCT5(){
+
+        List<FeedbackParticipantType> showResponseTo = new ArrayList<>();
+        showResponseTo.add(RECEIVER);
+
+        FeedbackQuestionAttributes question = FeedbackQuestionAttributes.builder()
+                .withShowResponsesTo(showResponseTo)
+                .withRecipientType(FeedbackParticipantType.OWN_TEAM_MEMBERS)
+                .withGiverType(STUDENTS)
+                .build();
+        boolean response = FeedbackResponsesLogic.inst().isResponseOfFeedbackQuestionVisibleToStudent(question);
+        assertEquals(response, true);
+    }
+    @Test
+    public void testIsResponseOfFeedbackQuestionVisibleToStudentCT6(){
+
+        List<FeedbackParticipantType> showResponseTo = new ArrayList<>();
+        showResponseTo.add(RECEIVER);
+
+        FeedbackQuestionAttributes question = FeedbackQuestionAttributes.builder()
+                .withShowResponsesTo(showResponseTo)
+                .withRecipientType(FeedbackParticipantType.GIVER)
+                .withGiverType(STUDENTS)
+                .build();
+        boolean response = FeedbackResponsesLogic.inst().isResponseOfFeedbackQuestionVisibleToStudent(question);
+        assertEquals(response, true);
+    }
+
+    @Test
+    public void testIsResponseOfFeedbackQuestionVisibleToStudentCT7(){
+
+        List<FeedbackParticipantType> showResponseTo = new ArrayList<>();
+        showResponseTo.add(RECEIVER_TEAM_MEMBERS);
+
+        FeedbackQuestionAttributes question = FeedbackQuestionAttributes.builder()
+                .withShowResponsesTo(showResponseTo)
+                .withRecipientType(NONE)
+                .withGiverType(NONE)
+                .build();
+        boolean response = FeedbackResponsesLogic.inst().isResponseOfFeedbackQuestionVisibleToStudent(question);
+        assertEquals(response, false);
+    }
+
+    @Test
+    public void testIsResponseOfFeedbackQuestionVisibleToStudentCT8(){
```

Comment:
let's rename the test cases to be more descriptive as well, following the convention testIsResponseOfFeedbackQuestionVisibleToStudent_<what scenario the test case is testing>_<the expected result>, an example is above  on line 1687 `testGetSessionResultsForUser_orphanResponseInDB_shouldStillHandleCorrectly`

Reply:
corrected test case descriptions